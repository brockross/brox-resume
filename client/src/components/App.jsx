import React from 'react';

import Header from './Header.jsx';
import SweApplications from './SweApplications.jsx';
import ProfessionalXp from './ProfessionalXp.jsx';
import Personal from './Personal.jsx';
import Skills from './Skills.jsx';
import UsedInApp from './UsedInApp.jsx';

import sampleData from '../dummyData.js';

class App extends React.Component {
  constructor() {
    super();

    this.state = {
      sweApps: JSON.parse(sampleData)
    }
  }

  render(){
    return(
      <div>
        <Header/>
        <SweApplications sweApps={this.state.sweApps}/>
        <ProfessionalXp/>
        <Personal/>
        <Skills/>
        <UsedInApp/>
      </div>
    )
  }
}

export default App;