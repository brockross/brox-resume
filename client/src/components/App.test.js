import React from 'react';
import { shallow } from 'enzyme';

import App from  './App.jsx';

describe('MyComponent', () => {
  it('should render correctly', () => {
    const component = shallow(<App/>);

    expect(component).toMatchSnapshot();
  });
});