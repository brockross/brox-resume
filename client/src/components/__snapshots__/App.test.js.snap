// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render correctly 1`] = `
<div>
  testing woot woot
</div>
`;
